README.adoc

```diff
@@ -180,9 +180,10 @@ in `InventoryManager.java`. You will look into this class in more detail in the
 
 === Adding the @Retry and @Fallback annotations
 
-Our `inventory` microservice is now able to recognise that the `system` microservice has been taken
-down for maintenance through the 503 HTTP response returned. As a result, it has thrown an IOException.
-A fallback method now needs to be set out to deal with this failure.
+Because of the 503 HTTP response that was returned,
```

Comment:
What if we gave "because of the 503 HTTP response" its own sentence? For example, "The server returns the 503 HTTP response. The inventory microservice is now able to recognize that the system microservice was taken down for maintenance."

Reply:
Ah got the text box back after I submit all the reviews; interesting GitHub experiences. Looking at the full context of this statement, it's the start of a new section. By saying "The server returns the 503 HTTP response." as the first sentence in this section is not clear.

Reply:
OK, we can stick with what you had, "The `inventory` microservice is now able to recognize that the `system` microservice was taken down for maintenance because of the 503 HTTP response."